cmd/platform/ldap.go

```diff
@@ -32,10 +33,11 @@ func ldapSyncCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if ldapI := einterfaces.GetLdapInterface(); ldapI != nil {
-		if err := ldapI.Syncronize(); err != nil {
-			CommandPrintErrorln("ERROR: AD/LDAP Synchronization Failed")
+		job, err := ldapI.StartSynchronizeJob(true)
+		if err != nil || job.Status == model.JOB_STATUS_ERROR {
```

Comment:
Maybe this should check for "cancelled" as  well? In fact, I'm wondering if the jobs package could benefit from some utility functions like `didFinishSuccessfully()` and `didFinishUnsuccessfully()` and `hasNotFinishedYet()` etc. These would then ensure that whereever in the code we do checks like this, the different statuses are treated consistently.

Reply:
Yup, I can add cancelled.